src/config/config.ts

```diff
@@ -104,8 +104,9 @@ export class Configuration {
   cardanoAddressFormat = 'stake[a-z0-9]{54}';
   defichainAddressFormat =
     this.environment === Environment.PRD ? '8\\w{33}|d\\w{33}|d\\w{41}' : '[78]\\w{33}|[td]\\w{33}|[td]\\w{41}';
+  railgunAddressFormat = '0zk[a-z0-9]{1,124}';
 
```

Comment:
Signature format is the same as Ethereum?

Reply:
no, it is not, this one starts with **0zk** and the length can be up to 128 long

Reply:
Please forget the previous answer. Yes, signature is similar than Ethereum but it needs a different algorithm for verification. Addresses are different.

Reply:
Ok, interesting 👍